src/app/layout.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Info } from 'react-feather';
+import Image from 'next/image';
+import { Space_Grotesk } from 'next/font/google';
+import Logo from '../../public/logo.svg';
+import { ExternalLink } from '../components/ExternalLink';
+
+import 'react-tooltip/dist/react-tooltip.css';
+import './global.css';
+import { Form } from '../components/Form';
+
+const spaceGrotesk = Space_Grotesk({
+  weight: ['400', '600', '700'],
```

Comment:
Already posted earlier. This should be handled globally / through css.

Reply:
The font files are downloaded at build time so there's no request from the stylesheet
we can choose to create a CSS variable from this font import and use it but since the whole app uses the same font, this font family should be applied once only anyway, **I would say we should keep it as it is**
https://beta.nextjs.org/docs/optimizing/fonts